modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,55 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+Logging into Keycloak gives you access to identity and access management.
+
+.Procedure
+
+. Log into the Keycloak Console after creating the Argo CD instance:
++
+[source,yaml]
+----
+kubectl -n argocd get route keycloak
+
+NAME   HOST/PORT                                                                PATH   SERVICES   PORT    TERMINATION   WILDCARD
+keycloak    keycloak-default.apps.ci-ln-******.origin-ci-int-aws.dev.**.com          keycloak        <all>   reencrypt     None
+----
+. Get the Keycloak credentials which is stored as environment variables in the Keycloak pod.
++
+[source,yaml]
+----
+  kubectl -n argocd get pods
+
+  NAME                                         READY   STATUS             RESTARTS   AGE
+  keycloak-1-2sjcl                                  1/1     Running            0          45m
+----
+. Get the Keycloak username.
++
+[source,yaml]
+----
+  kubectl -n argocd exec keycloak-1-2sjcl -- "env" | grep SSO_ADMIN_USERNAME
+
+SSO_ADMIN_USERNAME=Cqid54Ih
+----
+. Get the Keycloak password.
++
+[source,yaml]
+----
+  kubectl -n argocd exec keycloak-1-2sjcl -- "env" | grep SSO_ADMIN_PASSWORD
+
+SSO_ADMIN_PASSWORD=GVXxHifH
+----
+. Enter the Keycloak username and password. Click *Log in*.
+
+[NOTE]
+====
+The Keycloak instance takes 2-3 minutes to install and run. You only see the option *LOGIN VIA KEYCLOAK* after the keycloak instance is up.
+====
+
+By default, any user logged into Argo CD has read-only access. You can manage the user level access by updating the `argocd-rbac-cm` configmap.
```

Comment:
I would put a colon at the end of this sentence since it precedes a code snippet.

`...the argocd-rbac-cm configmap:`